### ISSUE TITLE
Constraint warnings after pod update from 

### ISSUE THREAD

**Issue by RaiseyDaisie** [SwipeCellKit/SwipeCellKit#201]
Target Deployment: iOS 11.0
Xcode: 9.3.1
CocoaPods 1.5.2

Have just done a pod update which upgrades SwipeCellKit ...

`Installing SwipeCellKit 2.3.1 (was 2.2.0)`

Received an error that said my SwipeTableViewCellDelegate protocol was not implemented and so I  changed the following from:

```
func visibleTableViewRect() -> CGRect {
     return tableView.safeAreaLayoutGuide.layoutFrame
}
```
... to: 

```
func visibleRect(for tableView: UITableView) -> CGRect? {
        return tableView.safeAreaLayoutGuide.layoutFrame
    }
```

Unfortunately now on running my application and attempting to swipe, I receive the following warnings ... how do I get these removed?
```
Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. 
	Try this: 
		(1) look at each constraint and try to figure out which you don't expect; 
		(2) find the code that added the unwanted constraint or constraints and fix it. 
(
    "<NSLayoutConstraint:0x600000489c40 SwipeCellKit.SwipeActionButtonWrapperView:0x7ff67053b6e0.height >= 66   (active)>",
    "<NSLayoutConstraint:0x60000048aff0 V:|-(0)-[SwipeCellKit.SwipeActionButtonWrapperView:0x7ff67053b6e0]   (active, names: '|':SwipeCellKit.SwipeActionsView:0x7ff670433090 )>",
    "<NSLayoutConstraint:0x60000048bd60 SwipeCellKit.SwipeActionButtonWrapperView:0x7ff67053b6e0.bottom == SwipeCellKit.SwipeActionsView:0x7ff670433090.bottom   (active)>",
    "<NSLayoutConstraint:0x60000048cc60 SwipeCellKit.SwipeActionsView:0x7ff670433090.height == Homie.SummaryHeaderCell:0x7ff6710d2800'SummaryHeaderCell'.height   (active)>",
    "<NSLayoutConstraint:0x604000491210 'UIView-Encapsulated-Layout-Height' Homie.SummaryHeaderCell:0x7ff6710d2800'SummaryHeaderCell'.height == 59.5   (active)>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x600000489c40 SwipeCellKit.SwipeActionButtonWrapperView:0x7ff67053b6e0.height >= 66   (active)>

Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
```

**Comment by kurabi** [comment]
Implementing `visibleRect(for tableView: UITableView) -> CGRect?` is optional and I put in a fix for this in v2.3.2 which should be out shortly.

Delete this method implementation on your end unless you are using tall cells (see Advanced.md).

**Comment by RaiseyDaisie** [comment]
Updated to 2.3.2 and removed visibleRect implementation. All good :) Thanks so much for the quick response.
